src/Nethermind/Nethermind.Merge.Plugin/Synchronization/StartingSyncPivotUpdater.cs

```diff
@@ -13,15 +13,14 @@
 using Nethermind.Db;
 using Nethermind.Logging;
 using Nethermind.Merge.Plugin.Handlers;
-using Nethermind.Serialization.Rlp;
 using Nethermind.State.Snap;
 using Nethermind.Synchronization;
 using Nethermind.Synchronization.ParallelSync;
 using Nethermind.Synchronization.Peers;
 
 namespace Nethermind.Merge.Plugin.Synchronization;
 
-public class PivotUpdator
+public class StartingSyncPivotUpdater
```

Comment:
`StartingSyncPivotUpdater` is a confusing name compared to `PivotUpdator`, not sure what `StartingSync` means in this context

Reply:
Its 'Starting' - 'SyncPivot' - 'Updater'.